<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Part1</class>
 <widget class="QWidget" name="Part1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblTitleP1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Face with OpenCV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChooseP1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTDisplayP1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>261</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDetectP1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Detect Face</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cascade Xml</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbP1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>haarcascade_frontalcatface.xml</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>haarcascade_frontalcatface_extended.xml</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>haarcascade_frontalface_alt.xml</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>haarcascade_frontalface_alt2.xml</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>haarcascade_frontalface_alt_tree.xml</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>haarcascade_frontalface_default.xml</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
